mycroft_admin_service.py

```diff
@@ -168,13 +168,22 @@ def ssh_disable(*_):
 
 
 def reset_system(*_):
+    # Remove all skills except Pairing (which is needed after wipe)
     call("""mkdir /opt/mycroft/safety &&
```

Comment:
Just in case something messes up, how about making this `mkdir -p /opt/mycroft/safety` in case it already exists.

Reply:
Good idea.